app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -84,3 +91,47 @@ def get_all_records():
         }
         send_api_error_slack_notif(body, data, error=e, request_info=request_info, channel='#dev-logging-etl')
         return request_info
+
+def ingest_sample_frame_goi_filter_options():
+    url = "https://api.airtable.com/v0/appFfYIupTrVQ0HJx/Sample%20Frame%20GOI?fields=Order&fields=Name&sort%5B0%5D%5Bfield%5D=Order&fields=French%20Name"
```

Comment:
What do you think about storing these magic constants more nicely, maybe there's a constants file we can use to store all URLs

Reply:
definitely a good idea! doesn't look like there's a constants file for all URLs, but i was thinking i could add this Airtable URL to our `config.py`?

Reply:
@annoyingpr can you add a comment to this function overall describing what it does? i.e. what sample frame goi filter options is